<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QRadioButton" name="radio1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>巨无霸</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>麦辣鸡腿堡</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>双脆牛肉堡</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>89</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>薯条</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>麦辣鸡翅</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>鸡块</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio7">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>130</y>
     <width>89</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>可乐</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio8">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>160</y>
     <width>89</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>雪碧</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>汉堡</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>小食</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>100</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>饮料</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
